### PR TITLE
Adding post "how to delete you master branch"

### PR DESCRIPTION
**Pull request by diemol**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

Adding a blog post that relates how a project can get rid of `master` as the default branch in a git project.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [ ] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: site_source_files/content/blog/2020/how_to_delete_your_master_branch.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+branch name to generate the docs. If you have scripts with similar purposes, check 
+them after changing the branch name.
+
+#### Open pull requests
+All of the tasks above could probably be achieved with a text editor and a massive, 
+but careful, “search and replace” across all the files in the repository. In short, 
+the process we followed to move to the new branch `trunk` was:
+
+1. Create a new branch called `trunk`, based on the `master` branch.
+1. Do all the items described in the previous points.
```
Comment (harsha509) [inline site_source_files/content/blog/2020/how_to_delete_your_master_branch.md:65]:
```suggestion
2. Do all the items described in the previous points.
```

File: site_source_files/content/blog/2020/how_to_delete_your_master_branch.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+1. Commit and push those changes.
```
Comment (harsha509) [inline site_source_files/content/blog/2020/how_to_delete_your_master_branch.md:66]:
```suggestion
3. Commit and push those changes.
```

**Comment by sindhudiddi** [comment]
Hi @diemol ,this looks interesting 
the numbering is not in order

File: site_source_files/content/blog/2020/how_to_delete_your_master_branch.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+1. Delete the `master` branch.
```
Comment (harsha509) [inline site_source_files/content/blog/2020/how_to_delete_your_master_branch.md:67]:
```suggestion
4. Delete the `master` branch.
```

**Comment by harsha509** [comment]
Hi @diemol ,

Few minor changes and we can merge it. This is much informative

Thanks,
Harsha

**Review by harsha509 — CHANGES_REQUESTED** [review]
Hi @diemol ,

Just the numbering and everything looks fine!

File: site_source_files/content/blog/2020/how_to_delete_your_master_branch.md (hunk 1 above)
Comment (diemol) [reply site_source_files/content/blog/2020/how_to_delete_your_master_branch.md:65]: @Harsha509 that is markdown, it is afterward rendered as a numbered list.
https://deploy-preview-510--jovial-austin-42fe02.netlify.app/blog/2020/how_to_delete_your_master_branch/

image

**Comment by diemol** [comment]
I'll merge this since the numbering is how markdown automatically creates the numbered lists.

File: site_source_files/content/blog/2020/how_to_delete_your_master_branch.md (hunk 1 above)
Comment (harsha509) [reply site_source_files/content/blog/2020/how_to_delete_your_master_branch.md:65]: :sweat_smile: Apologies. I was not aware of this.

**Pull request merged**